src/pages/quiz/summmary/QuizPage.tsx

```diff
@@ -69,7 +82,8 @@ const QuizPage: React.FC = () => {
                     radius={"sm"}
                     type="solid"
                     onPress={() =>
-                        navigation.navigate("QuestionsHolderScreen", {
+                        navigation.navigate("QuestionsScreen", {
+                            quizId: quiz.id,
```

Comment:
Według mnie nie powinnyśmy przekazywać tutaj nic poza quizId i pobierać pytania już po wejściu w ten ekran ale póki co to może zostać

Reply:
+1, ale to się poprawi przy okazji